shakedown/dcos/master.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for working with master"""
+
+from shakedown import *
+
+
+def partition_master(incoming=True, outgoing=True):
+    """ Partition Master's port alone. To keep DC/OS cluster running.
+    """
+
+    print('Partitioning master. Incoming:{} | Outgoing:{}'
+        .format(incoming, outgoing))
+
+    if incoming and outgoing:
+        copy_file_to_master("{}/partition_master_cmd"
+            .format(shakedown_dcos_dir()))
+        run_command_on_master("sh partition_master_cmd")
```

Comment:
That is along the lines of what I was thinking, yes, but then I just found this: https://github.com/dcos/shakedown/blob/622d0f4a03aef6def06fa4ce1dca4594dffe53c4/shakedown/dcos/agent.py#L54-L63

@sschneid, @kensipe -- is there a reason we chose to introduce bash scripts and copy them to the agents instead of running the commands in the Shakedown code itself?